internal/cache_options.go

```diff
@@ -47,14 +47,12 @@ func (c *KcpCacheOptions) GetCacheOptions() cache.Options {
 			},
 			&v1beta2.Kyma{}: {
 				Namespaces: map[string]cache.Config{
-					c.remoteNamespace: {},
-					c.kcpNamespace:    {},
+					c.kcpNamespace: {},
```

Comment:
The field `remoteNamespace string` can be removed from `KcpCacheOptions` as well, right?